CHANGELOG.md

```diff
@@ -21,6 +21,10 @@ Changelog
   [Graham Campbell](https://github.com/GrahamCampbell)
   [#167](https://github.com/bugsnag/bugsnag-laravel/pull/167)
 
+* Fixed certificate verification on some systems
```

Comment:
This description exactly matches how we described it when we fixed it upstream on bugsnag-php.